product.yml

```diff
@@ -367,6 +367,8 @@ bug_mapping:
       issue_component: Installer / Agent based installation
     ose-agent-installer-orchestrator-container:
       issue_component: Installer / Agent based installation
+    ose-agent-installer-utils:
```

Comment:
Could you suffix with `-container` to have it match https://brewweb.engineering.redhat.com/brew/packageinfo?packageID=84386?

Reply:
Acch, right, thanks :)